galleries.py

```
@@ -512,376 +537,175 @@ def _gallery(fileroot, product_ids, catalog, links,
+            main_links.append(element)
+        elif isinstance(element, list):
+            main_links.append(element[0])
 
-            if neighbors[2] is None:
-                f.write('next &gt; |\n')
-            else:
-                f.write('<a href="%s">next &gt;</a> |\n' % neighbors[2][0])
+    previous_link = None
+    for index, item in enumerate(links):
+        if isinstance(item, tuple):
+            (url, title, alt,) = item
```

Ruff (0.15.2)

[warning] Unpacked variable `title` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)